packages/e2e/cypress/e2e/dates.cy.ts

```diff
@@ -102,9 +102,11 @@ describe('Date tests', () => {
 
         cy.findAllByText('Loading chart').should('have.length', 0); // Finish loading
 
-        cy.contains(`What's our total revenue to date?`)
-            .parents('.react-grid-item')
-            .contains('400');
+        cy.get('.react-grid-layout').within(() => {
+            cy.contains(`What's our total revenue to date?`)
+                .parents('.react-grid-item')
+                .contains('400');
+        });
```

Comment:
@joaoviana was all the texts affected by adding `keepMounted`?

Reply:
yup! bc of the tiles in each popover filter 😅

Reply:
ah I see!